modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/model/ModelOps.scala

```diff
@@ -32,6 +33,16 @@ object ModelOps {
     case StepState.Paused     => "Paused"
   }
 
+  implicit val resourceShow: Show[Resource] = Show.shows[Resource] {
+    case Resource.TCS    => "TCS"
+    case Resource.Gcal   => "GCAL"
+    case Resource.Gems   => "GEMS"
+    case Resource.Altair => "Alt."
```

Comment:
Any reason to not use Altair as the display value?

Reply:
I just wanted a shorter label as if they are too wide the layout can be altered

Reply:
Ok. However I _think_  Alt. isn't a good option - check with science to see what they think.